sdks/java/core/src/main/java/org/apache/beam/sdk/options/PortablePipelineOptions.java

```diff
@@ -140,4 +140,12 @@ static String getEnvironmentOption(
   boolean getEnableHeapDumps();
 
   void setEnableHeapDumps(boolean enableHeapDumps);
+
+  @Description(
+      "If set, attempts to produce a pipeline compatible with this prior version of the Beam SDK."
+          + " See https://cloud.google.com/dataflow/docs/guides/updating-a-pipeline.")
```

Comment:
I think we actually have some discussions on dev@ and this also is Flink and Samza. Makes sense, basically. It isn't rigorously defined, but we do OK with it. And I think it is more or less runner independent.